src/components/price/index.jsx

```diff
@@ -6,8 +6,9 @@ import Sale from "./_children/Sale";
 const COMPONENT_CLASS_NAME = "c-price";
 
 const Price = ({ children, className, ...rest }) => {
-	const subComponents = Object.values(Price).map((subcomponentType) =>
-		Children.map(children, (child) => (child?.type === subcomponentType ? child : null))
+	const allowedTypes = Object.values(Price).map((subcomponentType) => subcomponentType);
```

Comment:
I think these two lines would benefit future developers with just a line or two of comments explaining how this is ensuring children render order.

Reply:
Very nice